datis.php

```diff
@@ -19,6 +19,7 @@
 $clouds = $decoded->getClouds(); //CloudLayer array
 $windShearAlerts = $decoded->getWindshearRunways();
 $type = $_GET['type'] ?? null;
+$adelv = $_GET['adelv'] ?? null; //use for QFE calculation
```

Comment:
Explain the variable in the comment.
e.g. `// airport elevation level in meter`